### DISCUSSION TITLE
Response Time Inconsistency, what version fixed this?

### DISCUSSION THREAD

**Discussion by jmueducn** [kiali/kiali#7599] [Q&A] [answered]
My kiali is now v1.32.0(relatively old), when a workload takes both http and grpc requests at the same time. And a choose to display the service node, the Response Time label displayed on the graph is the same (however, it is actually different). The question is: what version the bug is fixed?
![123](https://github.com/user-attachments/assets/11cb9676-0532-4cb8-8707-9f2c9f1c6b35)

**Comment by aljesusg** [comment, marked as answer]
Istio version?
We launched kiali 1.88 last week...that version is so old...

Verify the compatibility https://kiali.io/docs/installation/installation-guide/prerequisites/#version-compatibility

That version works until istio 1.9 (out of support)

**Reply by jmueducn** [reply]
istio version 1.16
thanks

**Reply by aljesusg** [reply]
So for 1.16

Min kiali 1.59.1	
max kiali 1.63.2	
Istio 1.16 is out of support. Avoid 1.62.0,1.63.0,1.63.1 due to a regression.

**Comment by jshaughn** [comment]
After updating to a recent version, please let us know if this is still a problem.

**Reply by jmueducn** [reply]
A version of 1.63.2 can solve the problem

**Reply by jshaughn** [reply]
Great, thanks for reporting back!
